packages/core/src/scp-cf/xsuaa-service.ts

```diff
@@ -145,22 +147,25 @@ export function fetchVerificationKeys(
 
 export function fetchVerificationKeys(
   xsuaaUriOrCredentials: string | XsuaaServiceCredentials,
-  clientId?: string,
+  clientIdOrJku?: string,
   clientSecret?: string
 ): Promise<TokenKey[]> {
+  // The case where the XsuaaServiceCredentials are given as object
   if (typeof xsuaaUriOrCredentials !== 'string') {
     return fetchVerificationKeys(
-      xsuaaUriOrCredentials.url,
+      clientIdOrJku || xsuaaUriOrCredentials.url,
```

Comment:
Wouldn't that have to be:
```suggestion
      clientIdOrJku || `${xsuaaUriOrCredentials.url}/token_keys`,
```

Reply:
And if yes, we could rename `xsuaaUriOrCredentials` to `tokenKeyUrlOrCredentials`

Reply:
Hm, I guess this makes sense if somebody uses it in the legacy way still working on old accounts. On new one it will only work with the JKU. I will add also a warning for users.